source/views/tasks_view.py

```diff
@@ -164,3 +166,14 @@ def get_statuses(self):
         statuses_data = self.status_schema.dump(statuses, many=True).data
         return jsonify({'statuses':statuses_data})
 
+
+    def get_users_per_task(self, id_task):
+        authorization(request.headers.get('Authorization', None))
```

Comment:
Two things here:

1) It's not "authorization" it's "authentication". The difference is that the second one validates user and passowrd and the first the permission you're attempting to use. In this case I guess it's authentication.

2) 

Instead of using the request.headers.get("Authorization"), you can change your authentication method using:
auth = request.authorization
and then using auth.username and auth.password

In this case you will avoid decoding base64 strings and matching the username and password. 

This requires to change your authorization/authentication class.